ILRepack/Steps/ReferencesFixStep.cs

```diff
@@ -58,7 +59,10 @@ public void Perform()
             }
             foreach (var r in targetAssemblyMainModule.Types)
             {
-                fixator.FixMethodVisibility(r);
+                //Visibility should not be changed for Methods of Types that are on the Primary Assembly,
+                //when internalize is true
+                if(!_options.Internalize || !_repackContext.PrimaryAssemblyMainModule.Types.Any(x=> x.FullName == r.FullName))
```

Comment:
The boolean logic doesn't seem right to me here. 

Shouldn't it be 
`if (_options.Internalize && !_repackContext.PrimaryAssemblyMainModule.Types.Any(x=> x.FullName == r.FullName))`

@titomor-outsystems